docs/HACKING.md

```diff
@@ -6,9 +6,10 @@
 * [Get Docker Compose](https://docs.docker.com/compose/install/)
 * [Get Node](https://nodejs.org/en/download/)
 
-Once you have Node, Docker, and Docker Compose,
-`cp services/registry/.env-example services/registry/.env`
-(and make any adjustments you like), `npm i`, and `npm start` (or `docker-compose up`).
+Once you have Node, Docker, and Docker Compose, `cp .env-example .env` in each
```

Comment:
Slight adjustment here.

Reply:
I've seen some success storing the database config and then loading the rest of the config from the database on server start. In both cases though, env vars become something to mentally juggle

Reply:
That might be nice! I've also had success with [node-config](https://github.com/lorenwest/node-config) at $day_job but it might be overkill for now.